native/src/main/resources/META-INF/native-image/io.ballerina.stdlib/graphql-native/resource-config.json

```diff
@@ -0,0 +1,17 @@
+{
+  "resources": {
+    "includes": [
+      {
+        "pattern": "\\Qgraphiql.html\\E"
+      }
+    ]
+  },
+  "bundles": [
```

Comment:
We can remove this bundle since this has been added from lang.runtime

Reply:
removed